lms/djangoapps/student_account/views.py

```diff
@@ -16,8 +17,14 @@
 from django.utils.translation import ugettext as _
 from django.views.decorators.csrf import ensure_csrf_cookie
 from django.views.decorators.http import require_http_methods
+from django.utils import timezone
 from django_countries import countries
 from edxmako.shortcuts import render_to_response
+from provider.oauth2.models import (
```

Comment:
You only need `AccessToken` and `RefreshToken`, both of which can fit on one line.